pymc_marketing/model_builder.py

```diff
@@ -516,8 +556,8 @@ def fit(
             The training input samples. If scikit-learn is available, array-like, otherwise array.
         y : array-like | array, shape (n_obs,)
             The target values (real numbers). If scikit-learn is available, array-like, otherwise array.
-        progressbar : bool
-            Specifies whether the fit progress bar should be displayed.
+        progressbar : bool, optional
+            Specifies whether the fit progress bar should be displayed. Defaults to True.
```

Comment:
In the docstrings, it says it defaults to `None` . Could you please clarify this :) ?

Reply:
I have "Defaults to True" in the description. What else did you have in mind?